images/nbserve/README.md

```diff
@@ -0,0 +1 @@
+This container redirects files to the renderer container to be built. Presumably does other things as well. Though this is less verified.
```

Comment:
nit: Maybe mention that it's the primary public entry point (all traffic passes through it).